bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -481,7 +483,7 @@ public async Task<IActionResult> Logout(string logoutId)
             if (orgUser.Status == OrganizationUserStatusType.Invited)
             {
                 // Org User is invited - they must manually accept the invite via email and authenticate with MP
-                throw new Exception(_i18nService.T("UserAlreadyInvited", email, organization.Name));
+                throw new Exception(_i18nService.T("UserAlreadyInvited", email, WebUtility.HtmlDecode(organization.Name)));
```

Comment:
Can we add this as a getter on the model?

Reply:
Excellent suggestion! I've added methods to output the Decoded values and used them everywhere where the value would be used for display purposes.